hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -597,6 +751,28 @@ private LookupFileResponse lookupFile(LookupFileRequest request,
         .build();
   }
 
+  @RequestFeatureValidator(
+      conditions = ValidationCondition.OLDER_CLIENT_REQUESTS,
+      processingPhase = RequestProcessingPhase.POST_PROCESS,
+      requestType = Type.LookupFile
+  )
+  public static OMResponse disallowLookupFileWithECReplicationConfig(
+      OMRequest req, OMResponse resp, ValidationContext ctx)
+      throws ServiceException {
+    if (!resp.hasLookupFileResponse()) {
+      return resp;
+    }
+    if (resp.getLookupFileResponse().getKeyInfo().hasEcReplicationConfig()) {
+      throw new ServiceException("Key is a key with"
+          + " Erasure Coded replication, which the client can not understand."
+          + " Please upgrade the client before trying to read the key info"
+          + " for" + req.getLookupFileRequest().getKeyArgs().getVolumeName()
```

Comment:
nit. Missing space in concatenation.

Reply:
oh that magical copy paste... thank you! done